### PR TITLE
Up the Casperjs waitTimeout to avoid test timeouts

### PR DESCRIPTION
**Pull request by schalkneethling**
@alexgibson r?

### CODE REVIEW

**Comment by alexgibson** [comment]
Other than the typo in the commit message, this is an r+ from me :cake:

File: tests/lib/helpers.js [hunk 1]
```diff
@@ -10,7 +10,6 @@ var config = require('../lib/config');
 function done() {
     casper.options.pageSettings.userAgent = config.userAgent.phantomjs;
     casper.options.viewportSize = config.viewport.desktop;
-    casper.options.waitTimeout = 5000;
```
Comment (alexgibson) [inline tests/lib/helpers.js]: Wait, this should probably remain and be changed to `casper.options.waitTimeout = 5000;`, for the case where we may set a test specific value?

Comment (schalkneethling) [inline tests/lib/helpers.js]: Indeed, good catch. For some reason my mind was thinking, meh, we'll never need a longer timeout ;p

Comment (alexgibson) [inline tests/lib/helpers.js]: I saw thinking somewhere we might want to set a shorter timeout :P

**Comment by schalkneethling** [comment]
@alexgibson updated

**Comment by schalkneethling** [comment]
hmmmm, I updated the commit message, not sure why it does not reflect over here. Perhaps when you click the big green button it will show.

File: tests/lib/helpers.js (hunk 1 above)
Comment (schalkneethling) [inline tests/lib/helpers.js]: Well of course, one might want to do that as well :)

**Pull request merged**